apps/src/lab2/views/dialogs/GenericPrompt.tsx

```diff
@@ -91,8 +91,8 @@ const GenericPrompt: React.FunctionComponent<GenericPromptProps> = ({
         confirm: {
           callback: () => handleConfirm?.(prompt),
           disabled:
-            requiresPrompt !== false &&
-            (Boolean(errorMessage) || !prompt.length),
+            Boolean(errorMessage) ||
+            (requiresPrompt === true && !prompt.length),
```

Comment:
This is unrelated to the rest of the change and was an opportunistic bug fix - the logic was wrong.

Previously, if the prompt.length was zero _and_ a prompt was not required, it'd be enabled. However, if there's an error condition, then it always needs to be disabled.

Reply:
Can we just write `requiresPrompt` rather than `requiresPrompt === true`?